store/src/main/java/StoreApp.java

```diff
@@ -0,0 +1,27 @@
+import java.util.ArrayList;
+import java.util.Map;
+
+public class StoreApp {
+
+    public static void main(String[] args) {
+
+        RandomStorePopulator populator = new RandomStorePopulator();
+        ArrayList<Product> products = populator.populateTheStore();
+        Store evroOpt = new Store(products);
+        evroOpt.printProducts(products); //print all products from myStore
+
+        String path = System.getProperty("user.dir") + "/store/src/main/resources/sort.xml";
+        DOMxmlReader reader = new DOMxmlReader();
+        Map<String, String> sortCondition = reader.XMLreader(path);
+
+        ArrayList<Product> sortedProducts = evroOpt.sortProduct(sortCondition, "price");
```

Comment:
according to the task, sorting should be done according to the whole config file, not just one item. So, create a solution that supports multiple comporators.